sloppy-deployment/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
```

Comment:
Which problem is getting solved here? sloppy-cli convert docker-compose.yml files to sloppy.yml via `run` command per default. Environment vars or env files gets evaluated too. For a conversion without api access you could consider the https://github.com/sloppyio/sloppose project.

Reply:
I did not realize that the CLI is capable of this ;)

Reply:
@malud Sometimes we need env-specific data that do not fit into env vars so we need a way to process more complex configs out of files. Like a list of 200 values that must be present in the sloppy config.